traffic_ops/testing/api/v4/regions_test.go

```diff
@@ -142,6 +159,36 @@ func SortTestRegions(t *testing.T) {
 	}
 }
 
+func SortTestRegionsDesc(t *testing.T) {
+
+	var header http.Header
```

Comment:
just so you know, `nil` is a valid `http.Header` value, so you can just pass `nil` if you don't need the header

Reply:
Fixed.